Firebase/DynamicLinks/FIRDynamicLinks+Private.h

```diff
@@ -30,11 +30,6 @@ FOUNDATION_EXPORT NSString *const kFIRDLVersion;
  */
 FOUNDATION_EXPORT NSString *const kFIRDLReadDeepLinkAfterInstallKey;
 
-/**
- * Label exceptions from FDL.
- */
-FOUNDATION_EXPORT NSString *const kFirebaseDurableDeepLinkErrorDomain;
```

Comment:
Should the definition be moved as well?

Reply:
Not necessarily.  This is a minimal change versus many changes if we start peeling the onion to clean up directory structure and error handling.